src/content/configuration/mode.md

```diff
@@ -164,7 +164,7 @@ var config = {
   //...
 };
 
-module.exports = (env, argv) => {
+module.exports = (env, argv) => ({
```

Comment:
The original version has no problem, only when we're returning an object should we wrap the `{}` with `()`.

Reply:
It's a SyntaxError